test/services/fileStorage/aws/awsMultipleS3.test.js

```diff
@@ -0,0 +1,120 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const { expect } = require('chai');
+const mockery = require('mockery');
+const { Configuration } = require('@schul-cloud/commons');
+const mockAws = require('./s3.mock');
+const logger = require('../../../../src/logger');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+
+chai.use(chaiHttp);
+
+describe('multple S3 AWS file storage strategy', () => {
+	let aws;
+
+	const options = {
+		schoolId: '0000d186816abba584714c5f',
+	};
+
+	const ShouldFail = new Error('It succeeded but should have returned an error.');
+
+	let configBefore = {};
+
+	const storageProviderService = app.service('storageProvider');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['storageProviderService' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5222818)